DonutsGo/DonutsGo.DataAccess/Storage.cs

```diff
@@ -0,0 +1,11 @@
+﻿using DonutsGo.DataAccess.Entities;
+
+namespace DonutsGo.DataAccess
+{
+    public static class Storage
+    {
+        public static List<Product> Products { get; set; } = new List<Product>();
```

Comment:
I guess this is still here as you are work in progress with having database storage, right? After that is in place this in memory usage will be removed?